src/AureliaDependenciesPlugin.ts

```diff
@@ -41,7 +48,7 @@ class ParserPlugin {
     hooks.evaluateIdentifier.tap("imported var.moduleName", TAP_NAME, (expr: Webpack.MemberExpression) => {
       if (expr.property.name === "moduleName" &&
           expr.object.name === "PLATFORM" &&
-          expr.object.type === "Identifier") {
+          String(expr.object.type) === "Identifier") {
```

Comment:
Why? What can `expr.object.type` be that is not a string but would convert to `"Identifier"`?

Reply:
OK, I see now that newer TS shows an error here.
A cast is a hack, the real problem lies in my poor type definition of `expr.object`. 
Would be nice to fix it there instead.

Same for the later occurances of `String(expr.object.type)`